main.tf

```diff
@@ -147,3 +170,7 @@ output "bucket" {
   value = aws_s3_bucket.bucket.bucket
 }
 
+# dynamodb table
+output "dynamodb_lock_table" {
+    value = var.dynamodb_state_locking ? aws_dynamodb_table.state_lock_table[0].id : null
```

Comment:
even though its identical, use name property to be explicit - since backend config requires name.

Reply:
Ah, good to know. Making the change.